docs/src/modules/operations/pages/cli/index.adoc

```diff
@@ -1,5 +1,5 @@
 = CLI
-:page-aliases: cli:index.adoc[]
```

Comment:
This grief only existed because the page-aliases were not written correctly and were lacking on the other CLI pages that moved in #324.